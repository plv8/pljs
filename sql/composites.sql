CREATE TYPE acomp AS (x int, y text, z timestamptz);
DO LANGUAGE pljs $$
  var jres = pljs.execute("select $1::acomp[]", [ [ { "x": 2, "z": null, "y": null } ] ]);
  pljs.elog(NOTICE,JSON.stringify(jres));
$$;
DROP TYPE acomp;
