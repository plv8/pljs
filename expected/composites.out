CREATE TYPE acomp AS (x int, y text, z timestamptz);
DO LANGUAGE pljs $$
  var jres = pljs.execute("select $1::acomp[]", [ [ { "x": 2, "z": null, "y": null } ] ]);
  pljs.elog(NOTICE,JSON.stringify(jres));
$$;
CREATE TYPE acomp AS (x int, y text, z timestamptz);
DO LANGUAGE PLV8 $$
  var jres = plv8.execute("select $1::acomp[]", [ [ { "x": 2, "z": null, "y": null } ] ]);
  plv8.elog(NOTICE,JSON.stringify(jres));
$$;
NOTICE:  [{"acomp":[{"x":2,"y":null,"z":null}]}]
